frontend/src/forms/CreateAccountFormDialog.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+// Redux
+import { useDispatch } from 'react-redux';
+import { postUserRequest } from '../state/user/userActions';
+
+// Form
+import { TextField } from 'mui-rff';
```

Comment:
To use material ui and react-final-form together you need to wrap the components. This library just does it for you. You can export all of the Material UI wrappers from this library `mui-rff`:

`{ TextField, Checkbox, Radio, Select }` ... etc.